core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.spec.ts

```diff
@@ -732,6 +741,8 @@ describe('Conversation skin component', () => {
     componentInstance.displayedCard = displayedCard;
 
     componentInstance.ngOnInit();
+    windowRef.nativeWindow.onresize(null);
+    tick(100);
```

Comment:
Why is the `tick` needed at the end, if it is just for coverage should we add some comment explaining that so that someone doesn't remove this by accident? Ditto elsewhere

Reply:
The tick has been used here to handle flake in the front end.

Reply:
comment has been added #15859 PTAL